### PR TITLE
MP-5706 How to upgrade Spryker instance to Marketplace.

### PR DESCRIPTION
**Pull request by az-joss**
## PR Description

- Ticket: https://spryker.atlassian.net/browse/MP-5706

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/marketplace/dev/howtos/how-to-upgrade-spryker-instance-to-marketplace.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: "How-To: Upgrade Spryker instance to Marketplace"
+description: This article provides details how to upgrade Spryker instance to Marketplace
+template: howto-guide-template
+---
+
+This articles provides details how to upgrade existing instance of Spryker shop to Marketplace
```
Comment (asmarovydlo) [inline docs/marketplace/dev/howtos/how-to-upgrade-spryker-instance-to-marketplace.md:7]: Note for the feature please use text validation tools, e.g. Grammarly
```suggestion
This article provides details on how to upgrade the existing instance of Spryker Shop to Marketplace
```

File: docs/marketplace/dev/howtos/how-to-upgrade-spryker-instance-to-marketplace.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+## 1) Add core features
+
+Follow feature integration guids from the table, that provides basic and consistent functionality of Marketplace
```
Comment (asmarovydlo) [inline docs/marketplace/dev/howtos/how-to-upgrade-spryker-instance-to-marketplace.md:11]:
```suggestion
Follow feature integration guides from the table that provides the basic and functionality of Marketplace.
```

File: docs/marketplace/dev/howtos/how-to-upgrade-spryker-instance-to-marketplace.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+| Feature | Description |
+| --- | --- |
+| Marketplace Product Options <!--- LINK --> | Allows merchants to create their own product option groups and values. |
+| [Marketplace Merchant Opening Hours](/docs/marketplace/dev/feature-integration-guides/202108.0/merchant-opening-hours-feature-integration.html) | Merchant working/opening hours management. |
+| [Marketplace Promotions & Discounts](/docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-promotions-discounts-feature-integration.html) | Merchant products promotions and discounts management. |
+
+## 2) Add MerchantPortal features
+
+Follow feature integration guids from the table, that provides functionality for MerchantPortal Application
```
Comment (asmarovydlo) [inline docs/marketplace/dev/howtos/how-to-upgrade-spryker-instance-to-marketplace.md:38]:
```suggestion
Follow feature integration guides from the table that provides functionality for MerchantPortal Application.
```

File: docs/marketplace/dev/howtos/how-to-upgrade-spryker-instance-to-marketplace.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+| [Marketplace MerchantPortal Product Management](/docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-portal-core-feature-integration.html) | Allows to merchants manage their product in Merchant Portal. |
+| [Marketplace MerchantPortal Product Offer Management](/docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-portal-core-feature-integration.html) | Allows to merchants manage their product offers in Merchant Portal. |
+| Marketplace MerchantPortal Product Option Management <!-- LINK --> | Allows to merchants manage their product options in Merchant Portal. |
+| Marketplace MerchantPortal Order Management <!-- LINK --> | Allows to merchants manage their orders in Merchant Portal. |
+
+## 3) Build app
+
+To apply all changes regarding to database entities, data imports, search engine indexes, UI assets etc,  need rebuild your app.
+Depends on what virtualization solution do you use just follow recommendations bellow
```
Comment (asmarovydlo) [inline docs/marketplace/dev/howtos/how-to-upgrade-spryker-instance-to-marketplace.md:51]:
```suggestion
To apply all changes regarding database entities, data imports, search engine indexes, UI assets, etc. please rebuild your app.
Depending on what virtualization solution do you use follow the recommendations below.
```

File: docs/marketplace/dev/howtos/how-to-upgrade-spryker-instance-to-marketplace.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+Follow feature integration guids from the table, that provides functionality for MerchantPortal Application
+
+| Feature | Description |
+| --- | --- |
+| [Marketplace MerchantPortal Core](/docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-portal-core-feature-integration.html) | Allows to config server and the basic functionality of the Merchant Portal. |
+| [Marketplace MerchantPortal Product Management](/docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-portal-core-feature-integration.html) | Allows to merchants manage their product in Merchant Portal. |
+| [Marketplace MerchantPortal Product Offer Management](/docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-portal-core-feature-integration.html) | Allows to merchants manage their product offers in Merchant Portal. |
+| Marketplace MerchantPortal Product Option Management <!-- LINK --> | Allows to merchants manage their product options in Merchant Portal. |
+| Marketplace MerchantPortal Order Management <!-- LINK --> | Allows to merchants manage their orders in Merchant Portal. |
```
[the remark this answers is not in the document]
Comment (az-joss) [reply docs/marketplace/dev/howtos/how-to-upgrade-spryker-instance-to-marketplace.md:46]: correct and fixed, except:

  - Marketplace MerchantPortal Product Option Management

added link to module in github instead.

File: docs/marketplace/dev/howtos/how-to-upgrade-spryker-instance-to-marketplace.md (hunk 1 above)
Comment (az-joss) [reply docs/marketplace/dev/howtos/how-to-upgrade-spryker-instance-to-marketplace.md:7]: fixed

File: docs/marketplace/dev/howtos/how-to-upgrade-spryker-instance-to-marketplace.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+This articles provides details how to upgrade existing instance of Spryker shop to Marketplace
+
+## 1) Add core features
+
+Follow feature integration guids from the table, that provides basic and consistent functionality of Marketplace
+
+| Feature | Description |
+| --- | --- |
+| [Marketplace Merchant](/docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-feature-integration.html) | Merchant user management. |
+| [ACL](/docs/marketplace/dev/feature-integration-guides/202108.0/acl-feature-integration.html) | Merchant security management. |
```
[the remark this answers is not in the document]
Comment (az-joss) [reply docs/marketplace/dev/howtos/how-to-upgrade-spryker-instance-to-marketplace.md:16]: moved

File: docs/marketplace/dev/howtos/how-to-upgrade-spryker-instance-to-marketplace.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+template: howto-guide-template
+---
+
+This articles provides details how to upgrade existing instance of Spryker shop to Marketplace
+
+## 1) Add core features
+
+Follow feature integration guids from the table, that provides basic and consistent functionality of Marketplace
+
+| Feature | Description |
```
[the remark this answers is not in the document]
Comment (az-joss) [reply docs/marketplace/dev/howtos/how-to-upgrade-spryker-instance-to-marketplace.md:13]: it is present (line 16)

File: docs/marketplace/dev/howtos/how-to-upgrade-spryker-instance-to-marketplace.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+| Marketplace Cart <!--- LINK --> | Allows you to include a "Notes" field on the cart page. Buyers can add notes to a particular item or the whole cart. |
+| [Marketplace Order Management](/docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-order-management-feature-integration.html) | Extends the default orders with merchants information. |
+| [Marketplace Shipment](/docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-shipment-feature-integration.html) | Allows to split order into several shipments based on the merchants from which the items were bought. Merchants can see their shipments only. |
+| [Marketplace Return Management](/docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-return-management-feature-integration.html) | Order returns management enhancement for OMS. |
+
+Optional features
+
+| Feature | Description |
+| --- | --- |
+| Marketplace Product Options <!--- LINK --> | Allows merchants to create their own product option groups and values. |
```
[the remark this answers is not in the document]
Comment (az-joss) [reply docs/marketplace/dev/howtos/how-to-upgrade-spryker-instance-to-marketplace.md:32]: added

File: docs/marketplace/dev/howtos/how-to-upgrade-spryker-instance-to-marketplace.md [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+| [Marketplace Product Offer](/docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-offer-feature-integration.html) | Allows a Merchant to sell the Product based on concrete product with own price, stock, status and etc. |
+| [Marketplace Product Offer Prices](/docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-offer-prices-feature-integration.html) | Allows define custom prices for product offers. |
+| [Marketplace Inventory Management](/docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-inventory-management-feature-integration.html) | Stock & Availability Management as well as Multiple Warehouse Stock Management for product offers and merchant products. |
+| [Marketplace Wishlist](/docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-wishlist-feature-integration.html) | Allows the customers to track and save merchant offers and products they wish to purchase through the wishlist. |
+| Marketplace Cart <!--- LINK --> | Allows you to include a "Notes" field on the cart page. Buyers can add notes to a particular item or the whole cart. |
```
[the remark this answers is not in the document]
Comment (az-joss) [reply docs/marketplace/dev/howtos/how-to-upgrade-spryker-instance-to-marketplace.md:23]: already merged, fixed

File: docs/marketplace/dev/howtos/how-to-upgrade-spryker-instance-to-marketplace.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (az-joss) [reply docs/marketplace/dev/howtos/how-to-upgrade-spryker-instance-to-marketplace.md]: added missed MP IG, the rest in the Tobi's PR

**Review by asmarovydlo — DISMISSED** [review]

**Review by gibmirmilch — APPROVED** [review]

**Pull request merged**